sci-misc/mendeleydesktop/mendeleydesktop-1.17.8-r1.ebuild

```diff
@@ -64,19 +64,25 @@ src_unpack() {
 }
 
 src_prepare() {
+	# patch launcher script
+	epatch "${FILESDIR}/${PN}-1.17_libdir.patch"
+	epatch "${FILESDIR}/${PN}-1.17_qt5plugins.patch"
+	epatch "${FILESDIR}/${PN}-1.17_unix-distro-build.patch"
```

Comment:
please move to `PATCHES`, put `default` here, and then I think we can also nix `eutils`